src/components/icons/plus.tsx

```diff
@@ -18,30 +18,24 @@ import React from 'react'
 
 import { IconProps } from './types'
 
-export const PlusIcon: React.FC<IconProps> = ({
-  width = 11,
-  height = 10,
-  viewBox = '0 0 11 10',
-  color = '#6E6E6E',
-  transform,
-}: IconProps) => (
+export const PlusIcon: React.FC<IconProps> = (props) => (
   <svg
-    width={width}
-    height={height}
-    viewBox={viewBox}
+    width="11"
+    height="10"
+    viewBox="0 0 11 10"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
-    transform={transform}
+    {...props}
   >
     <path
       fillRule="evenodd"
       clipRule="evenodd"
       d="M5.5 0.5C5.25147 0.5 5.05 0.701472 5.05 0.95V4.55H1.45C1.20147 4.55 1 4.75147 1 5C1 5.24853 1.20147 5.45 1.45 5.45H5.05V9.05C5.05 9.29853 5.25147 9.5 5.5 9.5C5.74853 9.5 5.95 9.29853 5.95 9.05V5.45H9.55C9.79853 5.45 10 5.24853 10 5C10 4.75147 9.79853 4.55 9.55 4.55H5.95V0.95C5.95 0.701472 5.74853 0.5 5.5 0.5Z"
-      fill={color}
+      fill="#6E6E6E"
```

Comment:
why do we not want components like that to be adjustable in terms of colors and sizes?

Reply:
Most of this should be possible using the passed `props`, and the rest can be done with styled-components. I don't think there is a need for some icons to work differently.